sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,161 @@
+                0,
+                0,
+                {
+                    "lot_id": line_id.lot_id.id,
+                    "product_id": line_id.lot_id.product_id.id,
+                    "product_uom_qty": line_id.quantity,
+                },
+            )
+            for line_id in self.line_ids
+        ]
+
+    def _action_open_sale_order(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_action_open_sale_order`

(ANN202)